### PR TITLE
feat(settings): Add backup recovery phone sub row compoment in Settings

### PR DESCRIPTION
**Pull request by vpomerleau**
## Because

* Create a new backup recovery phone sub row to be included in 2FA settings

## This pull request

* Creates the new component and adds stories, l10n and tests
* For storybook only, adds the new subrow to the 2FA unit row (only rendered if all subrow props passed in to the 2FA unit row component)

## Issue that this pull request solves

Closes: #FXA-10229

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [x] If applicable, I have modified or added tests which pass locally.
- [x] I have added necessary documentation (if appropriate).
- [x] I have verified that my changes render correctly in RTL (if appropriate).

## Screenshots (Optional)

image

image

image

image

Mobile example:

image

## Other information (Optional)

Note: phone numbers should always be rendered LTR, even in RTL languages

Will request UX and CX review as well, some content strings may change

### CODE REVIEW

File: packages/fxa-settings/src/components/Settings/SubRow/en.ftl [hunk 1]
```diff
@@ -12,4 +13,22 @@ tfa-row-backup-codes-get-new-cta = Get new codes
...
+# button to add/configure a backup recovery phone
+tfa-row-backup-phone-add-cta = Add
+# Button to remove a backup recovery phone from the user's account
+tfa-row-backup-phone-delete-button = Remove
+# Shown in tooltip on delete button or delete icon
+tfa-row-backup-phone-delete-title = Remove backup recovery phone
+tfa-row-backup-phone-delete-restriction = If you want to remove your backup recovery phone, add backup authentication codes or disable two-step authentication first to avoid getting locked out of your account.
+# "this" refers to backup recovery phone
+tfa-row-backup-phone-description = This is the easier recovery method if you canʼt use your authenticator app.
+tfa-row-backup-phone-sim-swap-risk-link = Learn about SIM swap risk
```
Comment (bcolsson) [inline packages/fxa-settings/src/components/Settings/SubRow/en.ftl:34]: There's a chance localizers might not understand `SIM swap`. Any chance we could include the link/explanation in a comment?

**Review by bcolsson — APPROVED** [review]
No issues in the string. One nice to have comment, but not blocking.

File: packages/fxa-settings/src/components/Settings/SubRow/index.tsx [hunk 2]
```diff
@@ -48,47 +69,103 @@ const SubRow = ({
...
+            <div className="flex items-center gap-1">
+              <StatusIcon />
+              {message}
+            </div>
+            {!localizedDescription &&
+              !localizedInfoMessage &&
+              linkExternalProps && <ExtraInfoLink />}
+          </div>
+          {localizedDescription && (
+            <p className="text-xs w-full mx-2 mt-2">
```
Comment (LZoog) [inline packages/fxa-settings/src/components/Settings/SubRow/index.tsx:115]: I know it makes sense to keep using `text-xs` until we update the font-size everywhere but it really is so small when there's a decent amount of copy 😅

**Review by LZoog — APPROVED** [review]
LGTM, excellent work on all of these states!! I resized them to look at mobile as well and all the spacing looks very nice and things seem to break to new lines where they should. 💎

File: packages/fxa-settings/src/components/Settings/SubRow/en.ftl (hunk 1 above)
Comment (vpomerleau) [reply packages/fxa-settings/src/components/Settings/SubRow/en.ftl:34]: We don't have the article yet, but I can add a short explanation in comment.

File: packages/fxa-settings/src/components/Settings/SubRow/index.tsx [hunk 3]
```diff
@@ -123,14 +200,109 @@ export const BackupCodesSubRow = ({
...
+  ) : (
+    <BackupRecoverySmsDisabledIcon className="-ms-1 -my-2 scale-50" />
+  );
+  const message = hasPhoneNumber ? (
+    // We will likely want to only retrieve the last 4 digits of the phone number from the backend
+    // but adding a slice here just in case to ensure only the last 4 digits are displayed
+    // u2022 is a bullet point character
+    // This format works for a North American phone number, but may need to be adjusted for other formats
+    // durring next phases of SMS feature rollout
+    // Phone numbers should always be displayed left-to-right, *including* in rtl languages
```
[the remark this answers is not in the document]
Comment (vpomerleau) [reply packages/fxa-settings/src/components/Settings/SubRow/index.tsx:236]: It's the same character - I'll use the bullet directly and note in comment what the unicode equivalent is.

File: packages/fxa-settings/src/components/Settings/SubRow/index.tsx (hunk 2 above)
Comment (vpomerleau) [reply packages/fxa-settings/src/components/Settings/SubRow/index.tsx:115]: You're right, it is very small - maybe best to switch to `text-sm`

**Pull request merged**